mkosi/distributions/arch.py

```diff
@@ -84,6 +84,10 @@ def repositories(cls, context: Context) -> Iterable[PacmanRepository]:
                 ) if repo in context.config.repositories
             ] + ["core", "extra"]
 
+            # the alarm repo needs to be at the top for Arch Linux ARM
+            if context.config.architecture.is_arm_variant():
+                repos.insert(0, "alarm")
```

Comment:
Please just initialize the `repos` list with `alarm` if needed instead of inserting to the front at the end. Also, can you post the contents of the default pacman.conf on alarm? I'd like to know if there are more repos we might need to take into account.

Reply:
Here's alarm's pacman.conf: https://github.com/archlinuxarm/PKGBUILDs/blob/master/core/pacman/pacman.conf

Reply:
Then adding it to the end is the way to go